sasmodels/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "sasmodels" %}
+{% set version = "0.98" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 151a0e3d75c1cf42df8126a3fa919e988b0f332051e876383739d50034716373
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv "
+  noarch: python
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - python
+    - numpy
+    - scipy
+    - bumps
+    - pyopencl
+
+test:
+  imports:
+    - sasmodels
+    - sasmodels.custom
+    - sasmodels.models
+
+about:
+  home: http://www.sasview.org
+  license: BSD
+  summary: Package for calculation of small angle scattering models using OpenCL
```

Comment:
Could you please also add a `description` here?

Reply:
A description is not absolutely needed.